crates/wash-lib/src/parser/mod.rs

```diff
@@ -370,7 +370,7 @@ impl DevManifestComponentTarget {
         if self
             .component_ref
             .as_ref()
-            .is_some_and(|v| v == component_ref)
+            .is_some_and(|v| component_ref.as_ref().is_some_and(|c| c == v))
```

Comment:
Super nit: couldn’t you flatten this or something?

Reply:
Something like that, I was just hoping to get it compiling but I should probably just change the `match` statement to pull the full image out